README.md

```diff
@@ -42,15 +56,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@ecekpo](https://github.com/ecekpo)
+- Twitter: [@ecekpo](https://twitter.com/ecekpo)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/ecekpo)
 
 ## 🤝 Contributing
 
```

Comment:
### Your MIT :memo:
![MIT-file (copy)](https://user-images.githubusercontent.com/93039359/186650269-453a730b-216e-4bc7-b6d9-278d7f5d6ef3.png)

- [ ] Your MIT.md file don't have your **name**, **current year**, and **app type**. Please add these things because this is the license file of your project so please take it seriously :thinking: